cmd/ctr/commands/images/import.go

```diff
@@ -64,6 +64,10 @@ If foobar.tar contains an OCI ref named "latest" and anonymous ref "sha256:deadb
 			Name:  "index-name",
 			Usage: "image name to keep index as, by default index is discarded",
 		},
+		cli.BoolFlag{
+			Name:  "all-platforms",
+			Usage: "imports content for all platforms",
```

Comment:
Change the Usage to with default?
  "imports content for all platforms, by default is false"